packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2764,6 +2764,15 @@ export default class Meeting extends StatelessWebexPlugin {
       this.recordingController.setUserPolicy(this.selfUserPolicies);
       this.controlsOptionsManager.setDisplayHints(this.userDisplayHints);
       this.handleDataChannelUrlChange(this.datachannelUrl);
+
+      Trigger.trigger(
+        this,
+        {
+          file: 'meetings',
+          function: 'setUpLocusInfoMeetingInfoListener',
+        },
+        EVENT_TRIGGERS.MEETING_INFO_AVAILABLE
+      );
```

Comment:
Shouldn't this be MEETING_INFO_UPDATED , are there any chance that meeting info get updated during the meeting as well ?

Reply:
can we also add some test